AWSAuthSDK/Sources/AWSAuthCore/AWSAuthCore.h

```diff
@@ -27,3 +27,6 @@ FOUNDATION_EXPORT const unsigned char AWSAuthCoreVersionString[];
 #import <AWSAuthCore/AWSSignInProvider.h>
 #import <AWSAuthCore/AWSSignInButtonView.h>
 #import <AWSAuthCore/AWSUIConfiguration.h>
+#import <AWSAuthCore/AWSAppleSignInProvider.h>
+#import <AWSAuthCore/AWSAppleSignInButton.h>
```

Comment:
It seems strange that we are putting SIWA-specific code in an `Core` module. That means customers who are only using AWSAuthSDK for their UserPools and no social providers will get SIWA classes for no reason?

Reply:
- Moved to separate SDK.
- Bought back the class from string logic since the AWSAuthUI doesnot depend on social provider sdks